### PR TITLE
Add Adeunis Devices

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
Adeunis Devices

Not contain examples and decoder files

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Review by Jaime-Trinidad — COMMENTED** [review]
profiles, images and yaml files from adeunis devices are added, examples and  decoder files were not provided yet.

File: vendor/adeunis/analog.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    min: -20
+    max: 75
+  # Relative humidity (fraction of 1)
+  relativeHumidity:
+    min: 0
+    max: 1
+
+# IP rating (optional)
+ipCode: IP67
+ipCode2: IP68
```
[the remark this answers is not in the document]
Comment (Jaime-Trinidad) [reply vendor/adeunis/analog.yaml:64]: Ok, there are some devices that offer both options, more in water-meters, this property is more informative right?

**Comment by Jaime-Trinidad** [comment]
Yes I am reaching out Adeunis for codec and examples files.

File: vendor/adeunis/analog.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (johanstokking) [reply vendor/adeunis/analog.yaml:64]: How do they offer both options? Aren't that two distinct devices then?

**Pull request merged**